notebooks/official/tabular_workflows/tabnet_on_vertex_pipelines.ipynb

```diff
@@ -170,7 +170,7 @@
       "source": [
```

Comment:
<p>To get the best set of hyperparameters for your dataset, you recommend running a HyperparameterTuningJob. -&gt;</p><p>To get the best set of hyperparameters for your dataset, it is recommended to run a HyperparameterTuningJob.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1378/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='53'/>